### DISCUSSION TITLE
正负号好像有点问题

### DISCUSSION THREAD

**Discussion by MeAPikachu** [physics-data/faq#86] [Q&A] [answered]
文档的Navier-Stokes的粘滞系数部分似乎正负号有点问题（粘滞系数的作用力 -- +  \nu \nabla^2 \vec U ）
<img width="575" alt="Screenshot 2021-08-30 at 11 21 56" src="https://user-images.githubusercontent.com/61261422/131280575-c114e372-f7e6-4895-ba4a-abb8c9c44d65.png">

但是在后面的求解的时候，似乎正负号又是正确的
<img width="565" alt="Screenshot 2021-08-30 at 11 21 48" src="https://user-images.githubusercontent.com/61261422/131280555-dff1c11a-7f9e-4641-b31a-2fa2a36a761a.png">

**Comment by Harry-Chen** [comment]
@CircuitCoder

**Comment by CircuitCoder** [comment, marked as answer]
是的，这是前部分的正负号确实有问题，是一个笔误，感谢！

（因为我是倒着写这个 PDF 的，非常 GG）

马上会发公告，对这个问题纠错。
